build_runner/lib/src/generate/watch_impl.dart

```diff
@@ -67,72 +61,50 @@ class WatchImpl {
       _expectedDeletes.add(id);
       if (existingOnDelete != null) existingOnDelete(id);
     };
+    builds = this.runWatch(until);
```

Comment:
nit: rename the instance method so you don't need `this` to disambiguate for the user (maybe just make it private?)

Reply:
Done